app/jenkins-for-jira-ui/src/components/ConnectJenkins/ConfigurationSteps/ConfigurationSteps.tsx

```diff
@@ -14,36 +14,86 @@ type AppProps = {
 const ConfigurationSteps = ({ currentStage }: AppProps) => {
 	const history = useHistory();
 
-	const stages: Stages = [
-		{
-			id: 'install',
-			label: 'Install plugin',
-			percentageComplete:
-				currentStage === 'install'
-					? 0
-					: 100,
-			status:
-				currentStage === 'install'
-					? 'current'
-					: 'visited',
-			href: '#',
-			onClick: () => { history.push('/install'); }
-		},
-		{
-			id: 'connect',
-			label: 'Connect your app',
-			percentageComplete: 0,
-			status:
-				currentStage === 'connect'
-					? 'current'
-					: 'unvisited',
-			href: '#'
-		}
-	];
+	const stages: { [key: string]: Stages } = {
+		install: [
+			{
+				id: 'install',
+				label: 'Install plugin',
+				percentageComplete: 0,
+				status: 'current',
+				href: '#',
+				onClick: () => { history.push('/install'); }
```

Comment:
/nit Can get rid of the curly bracers and `;` as it is a one liner.
```suggestion
				onClick: () => history.push('/install')
```

Reply:
Updated :)